pasture-core/Cargo.toml

```diff
@@ -38,4 +38,6 @@ name = "point_buffer_iterators_bench"
 harness = false
 
 [features]
-gpu = ["wgpu", "shaderc", "futures", "bytemuck"]
+default = ["gpu"]
```

Comment:
I don't think `gpu` should be a default feature, it pulls in all sorts of stuff that a regular user might not want